benchexec/tools/symbiotic4.py

```diff
@@ -46,24 +46,30 @@ def name(self):
         """
         return "symbiotic"
 
-    def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
+    def cmdline(self, executable, options, task, rlimits):
         """
         Compose the command line to execute from the name of the executable
         """
 
-        if propertyfile is not None:
-            options = options + ["--prp={0}".format(propertyfile)]
+        if task.property_file is not None:
```

Comment:
Can be simplified:
```suggestion
        if task.property_file:
```
Would be nice to use this consistently.

Reply:
fixed in 66eb7e734da93d400ad5e4745dbe7b45409bf451